README.md

```diff
@@ -137,34 +137,43 @@ A style guide for Sightly, the HTML templating system from Adobe Experience Mana
     </sly>
     ```
     
-  - [4.3](#4.3) <a name='4.3'></a> **Use meaningful identifier names.**
+  - [4.3](#4.3) <a name='4.3'></a> **Use naming conventions for identifiers.**
   
+    Sightly knows the following types of identifiers:
+    * defined in `data-sly-use` for a template library
+    * defined in `data-sly-template` for template names
+    * defined in `data-sly-use` for a JavaScript/Java object variable
+    * defined in `data-sly-list` for a list entry variable
+    * defined in `data-sly-test` for a test result variable
+    
+    To be able to distinguish the former two identifiers, the following naming conventions should be used:
+    * for template libraries: the prefix **lib-** followed by an arbitrary name
+    * for template names: the prefix **template-** followed by an arbitrary name
+    * for all other variable identifiers any name except the ones listed in http://docs.adobe.com/docs/en/aem/6-1/develop/sightly/global-objects.html can be used.
     This will enhance the readability of your Sightly scripts and and makes it easier for others to understand.
 
     ```html
-    <!--/* Bad */-->
-    <sly data-sly-use.comp="com.example.TeaserComponent">
-        ...
-    </sly>
-     
-    <!--/* Good */-->
-    <sly data-sly-use.teaser="com.example.TeaserComponent">
-        ...
+    <!--/* Java-object and template library */-->
+    <sly data-sly-use.teaser="com.example.TeaserComponent" data-sly-use.lib-teaser="teaser-templates.html">
+    	<!--/* template name> */-->
+        <sly data-sly-template.template-teaser1>...</sly>
+        <sly data-sly-call="template-teaser1"></sly>
+        <sly data-sly-call="lib-teaser.teaser3"></sly>
     </sly>
     ```
     
-  - [4.4](#4.4) <a name='4.4'></a> **Use camelcase for identifier names.**
+  - [4.4](#4.4) <a name='4.4'></a> **Use lower case for identifier names.**
   
-    Using camelCase will help to increase the readability of your identifiers.
+    Using lower case characters only will help to find identifiers in your code which are referenced from Sightly (e.g. in warnings). Although this might be harder to read than e.g. camel-case it is necessary because Sightly is internally converting all identifiers to lower-case.
```

Comment:
I'm not so convinced yet by this new guideline. Although the identifier names inside warnings are in lower case, I think you are reading identifier more often inside your Sightly scripts than you do in warnings messages. If you stick to the guideline by using camel case, you will also never have naming collisions.

Not using camel-case naming convention would really affect the readability too much in my opinion. In the following examples you can really see that it get's hard to read.

```html
<sly data-sly-use.productTeaserSmall="com.example.Class">
<sly data-sly-use.productteasersmall="com.example.Class">

<sly data-sly-use.mediaGalleryImage="com.example.Class">
<sly data-sly-use.mediagalleryimage="com.example.Class">
```

Do you have more use cases where you have to find identifier names in Sightly besides warnings? and does that really happen that often?